polkadot/runtime/westend/src/xcm_config.rs

```diff
@@ -172,8 +176,8 @@ pub type Barrier = TrailingSetTopicAsId<(
 			AllowTopLevelPaidExecutionFrom<Everything>,
 			// Subscriptions for version tracking are OK.
 			AllowSubscriptionsFrom<OnlyParachains>,
-			// Collectives and Fellows plurality get free execution.
-			AllowExplicitUnpaidExecutionFrom<CollectivesOrFellows>,
+			// Messages from system parachains or the Fellows plurality need not pay for execution.
+			AllowExplicitUnpaidExecutionFrom<SystemOrFellows<ParaId>>,
```

Comment:
very nit, it can be simplified also e.g.:
```suggestion
			AllowExplicitUnpaidExecutionFrom<(
						IsChildSystemParachain<ParaId>>,
						Fellows,   // <- SystemOrFellows -> Fellows can just to match collectives stuff
			)>,						
```